query/api/src/main/java/org/codice/ddf/admin/api/Field.java

```diff
@@ -48,5 +48,9 @@
     void pathName(String fieldName);
 
     void updatePath(List<String> subPath);
+
+    default T getSanitizedValue() {
```

Comment:
Can we call this`getSanitizedValue` in the graphql transformation classes instead of `getValue`? If so, I don't think we need the`HIDDEN_STRING` scalar type or the hidden coercing either. We could probably get rid of the `BaseHiddenField` class entirely.